themes-default/slim/src/components/helpers/show-selector.vue

```diff
@@ -21,11 +22,21 @@ import { mapState } from 'vuex';
 export default {
     name: 'show-selector',
     props: {
-        showSlug: String
+        showSlug: String,
+        followSelection: {
+            type: Boolean,
+            default: false
+        },
+        placeholder: String,
+        selectClass: {
+            type: String,
+            default: 'select-show form-control input-sm-custom'
```

Comment:
Should `selectClass` only be `select-show`?

Reply:
Sorry don't understand the question

Reply:
Since this will replace the classes won't `form-control` and `input-sm-custom` be needed to keep it looking like a form? I would have though just `select-show` needs to be replaced.